src/functionaltests/java/com/ericsson/ei/utils/HttpExecutor.java

```diff
@@ -15,20 +16,25 @@
 
 import java.io.IOException;
 
-public final class HttpExecutor {
+
+public class HttpExecutor {
 
     private static HttpExecutor instance;
     private CloseableHttpClient client = HttpClientBuilder.create().build();
     private static final Logger LOGGER = LoggerFactory.getLogger(HttpExecutor.class);
 
+    private HttpExecutor() {
```

Comment:
if you haven't others constructors, constructor by default will created automatically, so i think constructor is unneeded here

Reply:
I fixed this in another commit. Just set the class to final and remove the constructor. =)

Reply:
It is not my code. Comes from rebase and master branch.

Reply:
OK. Will rebase this PR before merge.